dashboard/modules/stats_collector/test_stats_collector.py

```diff
@@ -89,5 +90,82 @@ def getpid(self):
                 raise Exception(f"Timed out while testing, {ex_stack}")
 
 
+def test_memory_table(ray_start_with_dashboard):
+    assert (wait_until_server_available(ray_start_with_dashboard["webui_url"]))
+
+    @ray.remote
+    class ActorWithObjs:
+        def __init__(self):
+            self.obj_ref = ray.put([1,2,3])
+
+        def get_obj(self):
+            return ray.get(self.obj_ref)
+
+    my_obj = ray.put([1,2,3] * 100)
+    actors = [ActorWithObjs.remote() for _ in range(2)]
+    results = ray.get([actor.get_obj.remote() for actor in actors])
+    webui_url = format_web_url(ray_start_with_dashboard["webui_url"])
```

Comment:
Are we going to remove those tests on `test_metrics.py` in this case?

Reply:
Good question. I'll look and see about porting them over.